README.md

```diff
@@ -24,46 +31,85 @@ This section covers how to build and install the qseed application. Note that yo
 
 4.  Install the qseed application.
     ```bash
-    cmake --install .
+    sudo cmake --install .
+    ```
+
+# Quickstart using Thales HSMs 
```

Comment:
Andrew - Should we add a section to the README on how to use the provided dev containers, or is it expected that users won't use them for their quickstarts?

Reply:
The quickstart is targeted for administrators and they will typically set this up on a VM